### DISCUSSION TITLE
Cpu load spikes on the client

### DISCUSSION THREAD

**Discussion by Emanueleroc** [apohl79/audiogridder#417] [Help]
Hi everyone I'm new to audiogridder and I'm trying to figure out how to make it work best. I use an iMac 2015 as a client and an MSI prestige 15 as a server (although the problem I will tell you I have in both cases). In any case, once the server is recognized, after the third fourth audiogridder plugin inserted, the client cpu goes crazy and alternates between 100% and less. I read the settings I have, I don't know if they are right or I need to change something, please help me I would be very happy to be able to use the audiogridder correctly.. gr
![41069225-c093-4805-a9f8-1ac1d8b4284c](https://user-images.githubusercontent.com/79922368/109664316-4be6ca00-7b6d-11eb-9f95-dfc4f6df611a.jpg)
![7a9249df-fbd5-4050-bf20-ea46be8abf49](https://user-images.githubusercontent.com/79922368/109664317-4d17f700-7b6d-11eb-8c00-b3eb07d3bd95.jpg)
![0c815cae-ae69-4442-b3c4-f22adb9bab33](https://user-images.githubusercontent.com/79922368/109664318-4db08d80-7b6d-11eb-9c66-ecab491664c4.jpg)

azie

**Comment by Fittkenny** [comment]
quattro staggioni??

**Comment by CaBuffalo** [comment]
First of all, are you on LAN?

Second, you can try lower block settings.

On Tue, Mar 2, 2021 at 7:20 AM Fittkenny <notifications@github.com> wrote:

> quattro staggioni??
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/417#discussioncomment-421809>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6FGRNYQMEPMLBXUZF3TBT62FANCNFSM4YPDHSZQ>
> .
>


-- 
Kevin Ma
Supported by Gmail.

**Reply by Emanueleroc** [reply]
Yes, I am on the LAN

**Comment by Emanueleroc** [comment]
until now I had directly connected the two ethernet ports of the two PCs, they told me I was wrong. now i bought a gigabit network switch. I did well? what other advice can you give me?

**Comment by CaBuffalo** [comment]
Yes, you have to work in the LAN environment.

Wondering if you get any log / dump file when it happens.

And did you load more than 3 or 4 of different plugins? Does it work?

On Tue, Mar 2, 2021 at 2:02 PM Emanueleroc <notifications@github.com> wrote:

> until now I had directly connected the two ethernet ports of the two PCs,
> they told me I was wrong. now i bought a gigabit network switch. I did
> well? what other advice can you give me?
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/417#discussioncomment-423424>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6CSNBYV2CWJBD3PGV3TBVN65ANCNFSM4YPDHSZQ>
> .
>


-- 
Kevin Ma
Supported by Gmail.

**Reply by Emanueleroc** [reply]
is there any particular setting that I need to set?

**Comment by CaBuffalo** [comment]
No, but you can try different block settings. and the buffer setting in S1.

On Tue, Mar 2, 2021 at 2:54 PM Emanueleroc <notifications@github.com> wrote:

> is there any particular setting that I need to set?
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/417#discussioncomment-423560>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6CVICFSFTEN2JUE6ULTBVUEHANCNFSM4YPDHSZQ>
> .
>


-- 
Kevin Ma
Supported by Gmail.

**Reply by Emanueleroc** [reply]
I bought this switch, i set the ip parameters .. what should i do next? I just can't use it

![ee3e8bff-3ac9-4221-9ba8-75efd0d5c7b3](https://user-images.githubusercontent.com/79922368/109987071-98b1d880-7d06-11eb-99a6-35cf23355627.jpg)
![7b88ed09-b40b-4d0a-9b5e-b05e3916f2e9](https://user-images.githubusercontent.com/79922368/109987078-99e30580-7d06-11eb-982d-9040f1e66310.jpg)
![18aeff70-1aae-4a5b-aec8-d177b82779f3](https://user-images.githubusercontent.com/79922368/109987082-99e30580-7d06-11eb-85b1-ce0bda1047b7.jpg)
![e819f334-52ea-459f-92cb-39f25d7ec8f6](https://user-images.githubusercontent.com/79922368/109987083-9a7b9c00-7d06-11eb-8838-00f4567b83a0.jpg)

**Comment by CaBuffalo** [comment]
Your network setting looks good. It's gonna be the problem of S1. Can you
increase the audio buffer as well?

On Thu, Mar 4, 2021 at 7:29 AM Emanueleroc <notifications@github.com> wrote:

> I bought this switch, i set the ip parameters .. what should i do next? I
> just can't use it
>
> [image: ee3e8bff-3ac9-4221-9ba8-75efd0d5c7b3]
> <https://user-images.githubusercontent.com/79922368/109987071-98b1d880-7d06-11eb-99a6-35cf23355627.jpg>
> [image: 7b88ed09-b40b-4d0a-9b5e-b05e3916f2e9]
> <https://user-images.githubusercontent.com/79922368/109987078-99e30580-7d06-11eb-982d-9040f1e66310.jpg>
> [image: 18aeff70-1aae-4a5b-aec8-d177b82779f3]
> <https://user-images.githubusercontent.com/79922368/109987082-99e30580-7d06-11eb-85b1-ce0bda1047b7.jpg>
> [image: e819f334-52ea-459f-92cb-39f25d7ec8f6]
> <https://user-images.githubusercontent.com/79922368/109987083-9a7b9c00-7d06-11eb-8838-00f4567b83a0.jpg>
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/417#discussioncomment-429676>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6DHHY6S3IOJPOSLBOLTB6RMVANCNFSM4YPDHSZQ>
> .
>


-- 
Kevin Ma
Supported by Gmail.

**Reply by Emanueleroc** [reply]
I tried to run the audiogridder plugin and the audiogridder server on the same pc and it works fine, while if I connect it to the Mac server the cpu doesn't go crazy.

**Comment by phasegrid** [comment]
Ciao. Credo ti serva un cavo cross.

**Comment by Linxz84** [comment]
the same thing happens in my pro tools

**Reply by phasegrid** [reply]
Same here on 2 pc windows  7 cubase 10.5.
Don t know if could be addressed to lan card setup.

**Reply by phasegrid** [reply]
With a cross cable it works without router.
I ve been able to load more than 60 plugs.
But there is no stability.

**Comment by CaBuffalo** [comment]
Hi, guys,

Please follow the bug report procedure here. It's much easier for us to
help you out. Simply giving the symptom of the problem doesn't help too
much.

https://audiogridder.com/bug-reports/

Thanks.

On Wed, Mar 31, 2021 at 4:09 AM phasegrid ***@***.***> wrote:

> With a cross cable it works without router.
> I ve been able to load more than 60 plugs.
> But there is no stability.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/417#discussioncomment-552202>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6BGDSQKVJGFFUMYAULTGL7EXANCNFSM4YPDHSZQ>
> .
>


-- 
Kevin Ma
Supported by Gmail.

**Reply by phasegrid** [reply]
Thanks.
